src/components/Waveform.js

```diff
@@ -130,12 +94,16 @@ class Waveform extends Component {
             </Form>
           </Col>
         </Row>
-      </section>
+      </div>
     );
   }
 }
 
+const mapStateToProps = state => ({
+  peaksInstance: state.peaksInstance
+});
+
 export default connect(
-  null,
-  actions
+  mapStateToProps,
+  null
```

Comment:
No big deal, but you can omit this `null` as the 2nd argument for passing in `dispatch` stuff.